### ISSUE TITLE
Enable management of static users via the Tectonic console.

### ISSUE THREAD

**Issue by elsonrodriguez** [coreos/tectonic-forum#105]
## Feature Request
Enable management of static users via the Tectonic console.

### Environment
AWS
### Desired Feature
While evaluating Tectonic, in the absence of LDAP, it is useful to quickly and easily add users for evaluation.

Currently his requires modifying a ConfigMap and using an external hash-generation and id-generation tool. I've found this process to be error-prone.

Ideally these commands would be simply wrapped within the UI with a simple form, with a strong hint that LDAP is the recommended solution.

**Comment by mfburnett** [comment]
Hey @elsonrodriguez, thanks for this suggestion. We're continuously looking to improve how Tectonic works with user management, and we'll make sure to keep this suggestion in mind. Is this mostly just for an evaluation standpoint, or would you also be interested in having this be a long-term user management solution?

**Comment by elsonrodriguez** [comment]
It's for evaluation mostly. Kubernetes deployments in many organizations are still "Shadow IT", and plugging it into LDAP from the get-go is a non starter for those places.

However the basic user management would be able to get people over the hump easier, and would probably lead to higher Tectonic adoption.

**Comment by mfburnett** [comment]
@elsonrodriguez got it okay. Will bring this back to the product team and will update this issue with any progress on this. cc @robszumski

**Comment by abh** [comment]
For one of my installations we only have a couple users so managing them statically is fine, even preferred for simplicity.

**Comment by rhenretta** [comment]
Yeah, our AWS environment doesn't have LDAP configured, and I am now having to set it up for the sole purpose of kubernetes management.  I'd like to be able to simply use static users.

On a side note...  even better - support SAML so we can integrate with our SSO provider

**Comment by sym3tri** [comment]
SAML support will be available in Tectonic 1.6.2 which is coming out very soon.
